New Release 2.8.0

iliyaZelenko opened the pull request.

iliyaZelenko commented:
:tada: This PR is included in version 2.8.0 :tada:

The release is available on:
- [GitHub release](https://github.com/iliyaZelenko/vue-cool-select/releases/tag/v2.8.0)
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/vue-cool-select)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

Toby9876 commented:
Thank you for the "input-styles" prop addition. If possible it would be great to be able to add a CSS class in a similar way. But this will work for me.

If I may suggestion, I think the documentation (https://iliyazelenko.github.io/vue-cool-select/#/docs) could be updated to mention that inputElCustomAttributes will not work for all attributes so that you have a chance to understand why it is not working sometimes.

iliyaZelenko commented:
@Toby9876 I released a new version 2.9.0 in which you can use new `input-for-text-class` prop. For example:

```
input-for-text-class="my-class"
```

or 

```
:input-for-text-class="['my-class1', 'my-class2']"
```

or 

```
:input-for-text-class="{ 'my-class1': true, 'my-class2': myVariable }"
```

but I didn't test it at all 😄 

Hope this will work for you.

I also updated the documentation as you requested, moreover, I changed the design of the site.
